Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -33,7 +33,7 @@ class BrowserScrollingController: NSObject {
 
     weak var header: UIView?
     weak var footer: UIView?
-    weak var urlBar: URLBarView?
+    var urlBar: URLBarViewProtocol?
```

Comment:
Might be worth making `URLBarViewProtocol` a class protocol so we can keep this weak.